charts/nvidia-driver-runtime/templates/rbac.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "nvidia-driver-runtime.serviceAccountName" . }}
+  labels:
+    {{- include "nvidia-driver-runtime.labels" . | nindent 4 }}
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "nvidia-driver-runtime.serviceAccountName" . }}
+rules:
+- apiGroups: [""]
+  resources: ["nodes"]
+  verbs: ["get"]
```

Comment:
I know the get would be enough, but sometimes the list would also be used.
I would like to add all read verbs here, e.g. ["get", "list", "watch"]

Reply:
master PR: https://github.com/harvester/charts/pull/420